static/app/views/performance/trends/changeExplorer.tsx

```diff
@@ -164,9 +185,25 @@ function InfoItem({label, value}: {label: string; value: string}) {
 }
 
 function Header(props: HeaderProps) {
-  const {transaction, trendChangeType} = props;
+  const {
+    transaction,
+    trendChangeType,
+    trendView,
+    projects,
+    organization,
+    trendFunction,
+    trendParameter,
+  } = props;
 
   const regression = trendChangeType === TrendChangeType.REGRESSION;
+  const next = getTransactionSummaryLink(
```

Comment:
```suggestion
  const transactionSummaryLink = getTransactionSummaryLink(
```

Doesn't hurt to be explicit here, `next` didn't really tell me much about what the variable was for

Reply:
AHH truly embarrassing I forgot to change it before I committed 🥲